tests/expectations/core/test_expect_column_values_to_be_present_in_other_table.py

```diff
@@ -0,0 +1,156 @@
+import pandas as pd
+import pytest
+from contrib.experimental.great_expectations_experimental.expectations.expect_column_values_to_be_present_in_other_table import (
+    ExpectColumnValuesToBePresentInAnotherTable,  # noqa: F401 # needed for expectation registration
+)
+
+import great_expectations as gx
+import great_expectations.exceptions as gx_exceptions
+from great_expectations.compatibility.sqlalchemy_compatibility_wrappers import (
+    add_dataframe_to_db,
+)
+
+
+@pytest.fixture
+def referential_integrity_db(sa):
+    sqlite_engine = sa.create_engine(
+        "sqlite:///../../test_sets/referential_integrity_dataset.db"
+    )
+    order_table_1 = pd.DataFrame(
+        {
+            "ORDER_ID": ["aaa", "bbb", "ccc"],
+            "CUSTOMER_ID": [1, 1, 3],
+        }
+    )
+    order_table_2 = pd.DataFrame(
+        {
+            "ORDER_ID": ["aaa", "bbb", "ccc"],
+            "CUSTOMER_ID": [1, 5, 6],
+        }
+    )
+    customer_table = pd.DataFrame(
+        {
+            "CUSTOMER_ID": [1, 2, 3],
+        }
+    )
+
+    add_dataframe_to_db(
+        df=order_table_1,
+        name="order_table_1",
+        con=sqlite_engine,
+        index=False,
+        if_exists="replace",
+    )
+    add_dataframe_to_db(
+        df=order_table_2,
+        name="order_table_2",
+        con=sqlite_engine,
+        index=False,
+        if_exists="replace",
+    )
+    add_dataframe_to_db(
+        df=customer_table,
+        name="customer_table",
+        con=sqlite_engine,
+        index=False,
+        if_exists="replace",
+    )
+    return sqlite_engine
+
+
+def test_successful_expectation_run(referential_integrity_db):
```

Comment:
Could we add a marker? Seems like an integration test coupled with an SA dep? Same below.

Reply:
Done.